go.mod

```diff
@@ -5,7 +5,7 @@ go 1.14
 require (
 	github.com/coreos/go-semver v0.3.0
 	github.com/fatih/color v1.9.0
-	github.com/ghodss/yaml v1.0.0
+	github.com/ghodss/yaml v1.0.1-0.20190212211648-25d852aebe32
```

Comment:
I think this should be a proper version before merging.

Reply:
`go mod tidy` did this, I had `1.0.0` before and go-test CI failed, not sure what magic is happening there

Reply:
running go mod tidy again sorted it again, I guess some dependency magic